src/Microsoft.NetCore.Analyzers/Core/Performance/DoNotUseCountWhenAnyCanBeUsed.cs

```diff
@@ -251,18 +258,37 @@ public OperationActionsHandler(INamedTypeSymbol targetType, ImmutableHashSet<str
             public void AnalyzeInvocationOperation(OperationAnalysisContext context)
             {
                 var invocationOperation = (IInvocationOperation)context.Operation;
+                var method = invocationOperation.TargetMethod;
 
-                if (invocationOperation.Arguments.Length == 1)
+                if (invocationOperation.Arguments.Length == 1 &&
+                    IsEqualsMethod(method))
                 {
-                    var method = invocationOperation.TargetMethod;
-                    if (IsInt32EqualsMethod(method) &&
-                        (IsCountEqualsZero(invocationOperation, out var methodName) || IsZeroEqualsCount(invocationOperation, out methodName)))
+                    string operation;
```

Comment:
Nit: rename local to include `String` or `Str` suffix for clarity.

Reply:
Would you settle for `operationName`? I don't really like to suffix identifiers other than the expected ones (`Exception`,  `Attribute`,  `EventArgs`).

Reply:
I went with `operationKey`.